runtime/jvmMain/src/kotlinx/benchmark/JvmBenchmarkAnnotations.kt

```diff
@@ -10,8 +10,8 @@ actual typealias TearDown = org.openjdk.jmh.annotations.TearDown
 
 actual typealias Benchmark = org.openjdk.jmh.annotations.Benchmark
 
-@Suppress("NO_ACTUAL_CLASS_MEMBER_FOR_EXPECTED_CLASS")
-actual typealias BenchmarkMode = org.openjdk.jmh.annotations.BenchmarkMode
+@Target(AnnotationTarget.CLASS)
+actual annotation class BenchmarkMode(actual vararg val value: Mode)
```

Comment:
This workaround is temporary and will be fixed before going into master, right?
I found related open issue that will hopefully be fixed in 2.0: https://youtrack.jetbrains.com/issue/KT-61100

This workaround changes how benchmarks run in JVM. Basically, the annotation is ignored in JVM now.

Reply:
> This workaround changes how benchmarks run in JVM. Basically, the annotation is ignored in JVM now.

In these circumstances I'd suggest that we shouldn't merge this PR right now. I asked MPP team again about some design around this problem.